modules/default/calendar/calendar.css

```diff
@@ -19,5 +22,4 @@
 .calendar .time {
   padding-left: 30px;
   text-align: right;
-  vertical-align: top;
```

Comment:
Your solution, while nice, breaks my existing layouts: 

![Bildschirmfoto 2023-03-06 um 08 45 11](https://user-images.githubusercontent.com/1313547/223049255-ab78b151-e788-4cfb-90c5-fcd143434b81.png)

Probably we just need to add vertical-align top to the the title cell too (instead of the display flex). 

While looking at the code, the flex part in the symbols is for having them aligned to the right (useful for multiple icons in a row)

Reply:
so now I added only the missing align to the title column, failing tests are unrelated ...